public/app/plugins/panel/flamegraph/flamegraphV2/components/TopTable/FlameGraphTopTableContainer.tsx

```diff
@@ -107,12 +178,13 @@ function buildTableDataFrame(
   };
 
   for (let key in table) {
+    actionField.values.push(null);
```

Comment:
Seems weird maybe it would make sense to just pass what the custom renderer needs but in this case that would be 2 separate strings so I assume this would either need to be some JSON string or some awkward serialization transform.